packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts

```diff
@@ -85,9 +85,20 @@ export default ({ api, coreSagas, networks }) => {
   }
 
   const defineKycGoal = function * (search) {
+    // /#/open/kyc?tier={1, 2, ...}
     const params = new URLSearchParams(search)
-    yield put(actions.goals.saveGoal('kyc', { tier: params.get('tier') }))
-    yield put(actions.router.push('/login'))
+
+    const tier = params.get('tier') || 2
```

Comment:
we have tier types constants they are siting here
```
import { TIER_TYPES } from 'blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/model'
```

we can move them to some mor generic place but would help not to use magic numbers as it is 2 ;)